app/metrics.yaml

```diff
@@ -8169,6 +8176,61 @@ wallpapers:
     metadata:
       tags:
         - Wallpapers
+  onboarding_opened:
+    type: event
+    description: |
+      The wallpaper onboarding dialog has been displayed.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/26215
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/26893
+    notification_emails:
+      - android-probes@mozilla.com
+      - mavduevskiy@mozilla.com
```

Comment:
We only need to put our email down if the collection is permanent.  No need here.